packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1153,6 +1171,15 @@ class ThemeData with Diagnosticable {
   /// widgets.
   final DataTableThemeData dataTableTheme;
 
+  /// A theme for customizing the appearance and layout of [Checkbox] widgets.
```

Comment:
We probably need a sweeping change here - not needed for this PR - that makes it clear why the default value of these ThemeData properties is. Unless I'm missing something, please file an issue.

Reply:
https://github.com/flutter/flutter/issues/70935